driver-core/src/main/com/mongodb/internal/async/client/AsyncAggregateIterableImpl.java

```diff
@@ -57,18 +57,20 @@
     private String comment;
     private Bson hint;
 
-    AggregateIterableImpl(@Nullable final ClientSession clientSession, final String databaseName, final Class<TDocument> documentClass,
-                          final Class<TResult> resultClass, final CodecRegistry codecRegistry, final ReadPreference readPreference,
-                          final ReadConcern readConcern, final WriteConcern writeConcern, final OperationExecutor executor,
-                          final List<? extends Bson> pipeline, final AggregationLevel aggregationLevel, final boolean retryReads) {
+    AsyncAggregateIterableImpl(@Nullable final AsyncClientSession clientSession, final String databaseName,
```

Comment:
Most annoying part of this was that it caused lines to be too long so had to re-break many lines to get them back under the limit